### PR TITLE
Add commands to get and update log config

### LINKED ISSUE #115: Add command to update logging configuration
This was merged and released a litttle while ago: https://github.com/zwave-js/node-zwave-js/pull/1754

It would be great to add support for this in the server as well as in the Python library so we can add a service in Home Assistant to update the log level. This would eliminate one of the common reasons why users currently restart server.

### PR DESCRIPTION
**Pull request by raman325**
Related to #115 and zwave-js/node-zwave-js#1754. This will allow us to add support to get and update the log configuration in the Python library and then expose a UI in Home Assistant to get and update the log levels without having to restart the server.

Resolves #115

### CODE REVIEW

File: src/lib/outgoing_message.ts
Comment (balloob) [inline src/lib/outgoing_message.ts:35]: Extract this too .

File: src/lib/server.ts
Comment (balloob) [inline src/lib/server.ts:83]: Is everything returned from this function JSON serializable or should certain parts be manually converted (ie `Transport`)?

Comment (raman325) [reply src/lib/server.ts:83]: See my comment here: https://github.com/zwave-js/zwave-js-server/pull/116#discussion_r577374393

**Comment by marcelveldt** [comment]
Nothing is blocking this PR anymore right ?

**Comment by raman325** [comment]
Nope, forgot to switch it out of draft 😅

**Comment by marcelveldt** [comment]
this requires a new version of zwave-js right ? Should we bump the requirement ?

**Review by marcelveldt — APPROVED** [review]
Looks good!

**Comment by raman325** [comment]
Yes we will need a bump, the convention has been to bump the version in a separate PR

**Comment by marcelveldt** [comment]
Yes, so do the bump PR right after merging this because this one depends on it.

**Pull request merged**
